docs/user/environment/0-setup/elevation.rst

```diff
@@ -0,0 +1,66 @@
+Elevation
+=========
+
+The elevation of the launch site of a rocket is particularly important for
+determining its interaction with the atmosphere and Earth's gravitational field.
+
+With previously known elevation
+-------------------------------
+
+If the elevation, measured in meters above sea level, along with the latitude
+and longitude of the launch site is known, it can be used to initialize an
+Environment class instance as follows:
+
+.. jupyter-execute::
+
+    from rocketpy import Environment
+    
+    env = Environment(
+        date=(2019, 2, 10, 18), # year, month, day, hour
+        latitude=-21.960641,
+        longitude=-47.482122,
+        elevation=110,
+    )
+
+Using Open-Elevation API
+------------------------
+
+Fortunately, there are alternatives to find an approximate value of the
+elevation if the latitude and longitude values are known.
+
+One very useful and handy option is to use
+`Open-Elevation <https://open-elevation.com/>`_, a free and open-source
+elevation API.
+It is built right into RocketPy and can be used as follows.
```

Comment:
"It is built right into RocketPy" 

What does this mean? It gives me the notion that the API is built by RocketPy's team in the RocketPy repo.